viz_config.yml

```diff
@@ -3,7 +3,7 @@ storm_name: "Michael"
 stn_event: 287
 dates:
   start: '2018-10-09 00:00:00'
-  end: '2018-10-13 12:00:00'
+  end: '2018-10-13 05:00:00'
```

Comment:
looks from your GIFs like precip data is up to date with this new (earlier) time...?